src/Form/Type/Track.php

```diff
@@ -84,6 +84,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'entry_type' => YoutubeType::class,
             ]
         );
+
+        $builder->add(
+            'slug',
+            TextType::class,
+            [
+                'required' => false,
+                'label' => 'Short link',
+                'constraints' => new Regex([
+                    'pattern' => '/^[a-zA-Z0-9_\-]+$/',
```

Comment:
I don't see whitespace in this pattern.

Reply:
I think that "Momini-skali" looks better than "Momini%20skali" or "Momini+skali"